examples/connectivity/plot_adhd_covariance.py

```diff
@@ -5,10 +5,9 @@
 This example shows how to extract signals from regions defined by an atlas,
 and to estimate a covariance matrix based on these signals.
 """
- 
+n_subjects = 10  # Number of subjects to consider for group-sparse covariance
```

Comment:
Previously n_subjects wasn't used at all. I guess I just wanted to make sure that restricting to 10 subjects didn't change the plots too much.

Also a side-comment, this script uses n_jobs!=1 which means it won't work on Windows ...